app/components/Channels/ChannelCreateForm.js

```diff
@@ -32,7 +33,58 @@ import {
 const speeds = [TRANSACTION_SPEED_SLOW, TRANSACTION_SPEED_MEDIUM, TRANSACTION_SPEED_FAST]
 const defaultSpeed = TRANSACTION_SPEED_SLOW
 
+/**
+ * Animation to handle showing/hiding the amount fields.
+ */
+const ShowHide = Keyframes.Spring({
+  show: async next => {
+    await next({ pointerEvents: 'auto' })
+    await next({ opacity: 1 })
+  },
+  hide: { opacity: 0, pointerEvents: 'none' }
+})
+
+const FormButtons = ({ isNextButtonDisabled, nextButtonText, onBack }) => (
+  <Flex alignItems="center" mb={3}>
+    <Box width={1 / 4}>
+      <ChannelBackButton onClick={onBack} mr="auto" />
+    </Box>
+    <Flex width={2 / 4} justifyContent="center">
```

Comment:
Flex ratios do not add up to 1, but maybe it's intentional?

Reply:
Yes, this is deliberate, as I want the second box to be centerd, and there is nothing to display in the last 1/4.

This kinda illustrates what the layout does (See 2nd row)

```
[ - ][ - ][ - ][ - ]
[ - ][   -    ]
```